BaseApiBundle/Controller/BaseController.php

```diff
@@ -64,6 +69,11 @@ protected function reverseTransform(Request $request, $id, $type, $event, $event
             $em->persist($mixed);
             $em->flush();
 
+            if(in_array('OpenOrchestra\ModelInterface\Event\EventTrait\EventStatusable', class_uses($eventClass))) {
```

Comment:
@itkg-nanne : The point was about the hardcoded path. I'm not fan of that kinf of thing. Could it be possible to fire the event if ``nul !== $oldStatus`` for instance ?